app/views/application_records/show.haml

```diff
@@ -6,6 +6,10 @@ by:
 = @record.user.name_and_email
 = button_to 'Print this page',
   application_record_path(@record, format: :pdf), method: :get
+- if !@record.saved_for_later && @record.note_for_later.present?
```

Comment:
YUP THIS IS IT THIS IS HOW YOU DO IT
![screen shot 2016-10-03 at 4 34 09 pm](https://cloud.githubusercontent.com/assets/9645316/19053061/3e7f8e2e-8987-11e6-82d5-65af264e42f3.png)